Makefile

```diff
@@ -37,5 +37,11 @@ docker-sidecar:
 docker-testground:
 	docker build -t iptestground/testground:edge -f Dockerfile.testground .
 
-test:
+clean:
+	rm -rfi $${TESTGROUND_HOME}/plans/placebo
+	rm -rfi $${TESTGROUND_HOME}/plans/example
+
+test: install
+	testground plan import --from ./plans/placebo || true
+	testground plan import --from ./plans/example || true
```

Comment:
This looks a lot like the CircleCI tasks. Let's make CircleCI call the makefile: https://github.com/testground/testground/blob/master/.circleci/config.yml#L113-L116

Reply:
Yeah, I am planning on doing something about that, just not sure how exactly right now.